doc/data-files-description.md

```diff
@@ -15,7 +15,7 @@ This document contains information about all data files associated with this pro
 
 
 
-### current release (release-v11-20191126)
+### current release (release-v12-20191217)
```

Comment:
I think this is, too -- I'm going to revert these duplicate changes so that only the data release check list is being added.